packages/conform-dom/index.ts

```diff
@@ -16,12 +16,9 @@ export {
 export {
 	type FieldElement,
 	isFieldElement,
+	isGlobalInstance,
 	updateField as unstable_updateField,
 	createFileList,
-	createGlobalFormsObserver as unstable_createGlobalFormsObserver,
-	focus as unstable_focus,
-	change as unstable_change,
-	blur as unstable_blur,
```

Comment:
I moved all these exports to the future export so it's easier to manage as I put up more helpers that are only used by those future APIs.